design/src/components/Form.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useState } from 'react';
+
+const Form = () => {
+
+ const [name, setName] = useState('');
+ const [email, setEmail] = useState('');
+ const [address, setAddress] = useState('');
+ const [phone, setPhone] = useState('');
+
+ return (
+  <div>
+   <div className="img-overlay">
+   </div>
+   <div className="form-section">
+    <div className="form-wrapper">
+     <h3>Beställ smakprov på årets glögg</h3>
+     <input type="text" placeholder='Namn' id="name" value={name} onChange={(e) => setName(e.target.value)} /><br />
+     <input type="text" placeholder='E-mail' id="email" value={email} onChange={(e) => setEmail(e.target.value)} /><br />
```

Comment:
The input type for an email field could be set to `email` for better user experience